examples/example-gauth/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2018 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.googleAuth;
+
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.grpc.CallCredentials;
+import io.grpc.auth.MoreCallCredentials;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import com.google.pubsub.v1.PublisherGrpc;
+import com.google.pubsub.v1.ListTopicsResponse;
+import com.google.pubsub.v1.ListTopicsRequest;
+
+import java.io.FileInputStream;
```

Comment:
Here's our style guide http://go/java-style 
Imports should be sorted and no line break here.